src/jit/TableInl.h

```diff
@@ -0,0 +1,216 @@
+
+/*
+ * Copyright (c) 2022-present Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Only included by Backend.cpp */
+
+static void emit3Operands(sljit_compiler* compiler, Operand* params)
+{
+    JITArg srcArg;
+
+    for (int i = 0; i < 3; i++) {
+        operandToArg(params + i, srcArg);
+        sljit_emit_op1(compiler, SLJIT_MOV32, SLJIT_R(i), 0, srcArg.arg, srcArg.argw);
```

Comment:
MOVE_FROM_REG does not do anything if `srcArg.arg` is already SLJIT_R(i). There are similar cases below.